map/bookmark_manager.cpp

```diff
@@ -2818,6 +2822,12 @@ void BookmarkManager::MarksChangesTracker::OnUpdateMark(kml::MarkId markId)
     m_updatedMarks.insert(markId);
 }
 
+void BookmarkManager::MarksChangesTracker::OnUpdateLine(kml::TrackId lineId)
```

Comment:
Why OnUpdateLine instead of UpdateLine?

Reply:
This is the conventional naming style for that manager:
```cpp
    void OnAddMark(kml::MarkId markId);
    void OnDeleteMark(kml::MarkId markId);
    void OnUpdateMark(kml::MarkId markId);

    void OnAddLine(kml::TrackId lineId);
    void OnDeleteLine(kml::TrackId lineId);
    void OnUpdateLine(kml::TrackId lineId); <-- new method
```